extensions/adapters/vector/src/main/java/mil/nga/giat/geowave/adapter/vector/ingest/SimpleFeatureIngestOptions.java

```diff
@@ -30,11 +30,23 @@
 	@ParametersDelegate
 	private FeatureSerializationOptionProvider serializationFormatOptionProvider = new FeatureSerializationOptionProvider();
 
+	@ParametersDelegate
+	private GeometrySimpOptionProvider simpOptionProvider = new GeometrySimpOptionProvider();
+
 	@ParametersDelegate
 	private Object pluginOptions = null;
 
 	public SimpleFeatureIngestOptions() {}
 
+	public GeometrySimpOptionProvider getSimpOptionProvider() {
```

Comment:
a nit, but can "geom" or "geometry" be somewhere in the method names?  it seems like an important descriptor